src/main/scala/util/ExecuteOpenCL.scala

```diff
@@ -42,8 +61,8 @@ object ExecuteOpenCL {
     try {
       val src = writeToTempFile("code-", ".c", code).getAbsolutePath
       val bin = createTempFile("bin-", "").getAbsolutePath
-      val sources = s"$src $runtimePath/buffer_${buffer_impl}.c $runtimePath/ocl.c"
-      (s"clang -O2 $sources $includes -o $bin $libs -Wno-parentheses-equality" !!)
+      val sources = s"$src $runtimePath/buffer_$buffer_impl.c $runtimePath/ocl.c"
```

Comment:
its a bit hard to read that this is interpolating `buffer_impl`?

Reply:
For me, the syntax highlighting helps to clarify this